### DISCUSSION TITLE
Difficult to run nnn

### DISCUSSION THREAD

**Discussion by lngndvs** [jarun/nnn#1902] [Q&A] [answered]
I am hopeful that nnn will work to preview images, as well as general file management.  I have been an avid user of ranger; however, I think I am not smart enough to get image preview working anymore.  So I'm looking at nnn, and it looks nice... IF I can figure it all out.

The first problem is this: when I type "nnn" in a terminal on Manjaro i3, in kitty, or I think other terminal emulators, the following is printed on the console: "NNN_BMS."  And that is all.  After random googling over the years, I finally have been able to use nnn by typing "NNN_BMS='d:~/Downloads' nnn".  Only in that terminal.  When I set NNN_BMS to the same value in either /etc/environment or in .bashrc, still, nothing happens.  Finally, I have gotten it to work by defining an alias in .bashrc:  
   	alias n="NNN_BMS='d:Downloads' nnn"

If I then type "d" into nnn, this does not take me to that directory; rather, it toggles a more contextual version of a listing of the ~/ directory.

What am I missing?   Am I the only one experiencing this?  

Alan Davis

**Comment by N-R-K** [comment, marked as answer]
> the following is printed on the console: "NNN_BMS." And that is all.

This doesn't sound right. Defining `NNN_BMS` is optional and `nnn` should work fine without it. I also cannot reproduce this locally. Please provide more info about your setup, see the [issue template](https://github.com/jarun/nnn/blob/86d883eee4986c7125bd77415e1a9b6b94e6bb83/.github/ISSUE_TEMPLATE/bug_report.md?plain=1#L30) on what sort of information to provide.

> If I then type "d" into nnn, this does not take me to that directory

That's not how bookmarks work. You need to press `b` which will show you available bookmark keys, that's when you press `d`. See the wiki entry on bookmarks: https://github.com/jarun/nnn/wiki/Basic-use-cases#add-bookmarks

**Comment by N-R-K** [comment]
Closing due to no response.

**Comment by lngndvs** [comment]
My apologies are due, for not noticing this earlier.

nnn has never worked for me on my Manjaro i3wm setup.  Just now, I
installed nnn and ran it from a console.  Once again, as always, "NNN_BMS"
is output in the console.

After an hour or two of twiddling, I was able to get nnn to run on a
limited basis, but it's far too tricky for me to devote more time to it.

It has never worked.

Alan
